include/warcpp/warcpp.hpp

```diff
@@ -17,14 +17,13 @@ class Warc_Format_Error : public std::runtime_error {
 };
 
 class Warc_Record;
-std::istream &read_warc_record(std::istream &in, Warc_Record &record);
+void read_warc_record(std::istream &in, Warc_Record &record);
 
 //! A WARC record.
 class Warc_Record {
    private:
     std::string version_;
     Field_Map   warc_fields_;
-    Field_Map   http_fields_;
```

Comment:
Since we're removing HTTP fields, we should probably just use `fields_` for WARC fields. Same goes for `warc_content_length()` or `warc_field()`.